Trappist/src/Promact.Trappist.Test/TestConduct/TestConductRepositoryTest.cs

```diff
@@ -201,11 +199,98 @@ public TestConductRepositoryTest(Bootstrap bootstrap) : base(bootstrap)
             await _testConductRepository.RegisterTestAttendeesAsync(testAttendee, _stringConstants.MagicString);
             var attendeeId = await _trappistDbContext.TestAttendees.Where(x => x.Email.Equals(testAttendee.Email)).Select(x => x.Id).FirstOrDefaultAsync();
             //JSON string
-            var answer = "{ \"questionId\":1,\"optionChoice\":[1],\"code\":null,\"questionStatus\":1}";
+            var answer = "[{ \"questionId\":1,\"optionChoice\":[1],\"code\":null,\"questionStatus\":1}]";
             await _testConductRepository.AddAnswerAsync(attendeeId, answer);
             var attendeeAnswer = await _testConductRepository.GetAnswerAsync(attendeeId);
 
-            Assert.True(attendeeAnswer.QuestionId == 1 && attendeeAnswer.OptionChoice.Count() == 1 && attendeeAnswer.QuestionStatus == QuestionStatus.review);
+            Assert.True(attendeeAnswer.Count == 1);
+        }
+
+        /// <summary>
+        /// Test to get Test Attendee
+        /// </summary>
+        [Fact]
+        public async Task GetTestAttendeeByIdAsyncTest()
+        {
+            var testAttendee = InitializeTestAttendeeParameters();
+            await CreateTestAsync();
+            await _testConductRepository.RegisterTestAttendeesAsync(testAttendee, _stringConstants.MagicString);
+            var attendeeId = await _trappistDbContext.TestAttendees.OrderBy(x => x.Email).Where(x => x.Email.Equals(testAttendee.Email)).Select(x => x.Id).FirstOrDefaultAsync();
+            var attendee = await _testConductRepository.GetTestAttendeeByIdAsync(attendeeId);
+
+            Assert.NotNull(attendee);
+        }
+
+        /// <summary>
+        /// Test to check if Test Attendee exist
+        /// </summary>
+        [Fact]
+        public async Task IsTestAttendeeExistByIdAsyncTest()
+        {
+            var testAttendee = InitializeTestAttendeeParameters();
+            await CreateTestAsync();
+            await _testConductRepository.RegisterTestAttendeesAsync(testAttendee, _stringConstants.MagicString);
+            var attendeeId = await _trappistDbContext.TestAttendees.OrderBy(x => x.Email).Where(x => x.Email.Equals(testAttendee.Email)).Select(x => x.Id).FirstOrDefaultAsync();
```

Comment:
I don't have any clue on this (I read the code but couldn't find any validation for such requirement). Since its a unit testing `first` and `single` will work the same because both will fetch only one row from table.